applications/CompressiblePotentialFlowApplication/custom_elements/adjoint_base_potential_flow_element.h

```diff
@@ -140,30 +134,25 @@ class AdjointPotentialFlowElement : public Element
 
     void PrintData(std::ostream& rOStream) const override;
 
-   Element::Pointer pGetPrimalElement();
-
+    Element::Pointer pGetPrimalElement();
 
 protected:
 
-
     Element::Pointer mpPrimalElement;
 
-    void GetWakeDistances(array_1d<double,NumNodes>& distances);
```

Comment:
I remove here GetWakeDistances() because I now use the function from potential_flow_utilities